src/common/Telemetry/ProjectTelemetry.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <TraceLoggingProvider.h>
-#include <TraceLoggingDefines.h>
+#include "TraceLoggingProvider.h"
+#include "TraceLoggingDefines.h"
```

Comment:
Please change it back to "<  >" because otherwise it will break how we enable telemetry.